src/backend/version-tracker/version-tracker.lambda.ts

```diff
@@ -0,0 +1,167 @@
+import { metricScope, Unit } from 'aws-embedded-metrics';
+import type { Context } from 'aws-lambda';
+import * as _AWS from 'aws-sdk';
+// import * as AWSXRay from 'aws-xray-sdk-core';
+import * as aws from '../shared/aws.lambda-shared';
+import { requireEnv } from '../shared/env.lambda-shared';
+import { ENV_PACKAGE_DATA_BUCKET_NAME, ENV_PACKAGE_DATA_KEY_PREFIX, ENV_VERSION_TRACKER_BUCKET_NAME, ENV_VERSION_TRACKER_OBJECT_KEY, MetricName, METRICS_NAMESPACE } from './constants';
+
+
+const PACKAGE_PREFIX_REGEX = /^data\/((?:@[^/]+\/)?[^/]+)\/v([^/]+)\/$/;
+
+// Batch size that limits how many outgoing S3 calls are made at a time.
+// This can be tweaked as needed (increased if we want to squeeze out more
+// performance, or decreased if we are getting too throttled).
+const BATCH_SIZE = 200;
+
+export async function handler(event: any, context: Context) {
+  console.log(JSON.stringify(event, null, 2));
+  console.log(JSON.stringify(context, null, 2));
```

Comment:
```suggestion
  console.log(`Event: ${JSON.stringify(event, null, 2)}`);
```

It's not use to drop the `Context` object - it'll only contain information that you already know if you're staring at the CloudWatch Logs for an execution.